docs/agents/sematext-agent/containers/installation.md

```diff
@@ -21,13 +22,43 @@ sematext/agent:latest
 
 _[Read more](../permission-requirements.md#bind-mounts) about why Sematext Agent needs access to host files and directories._
 
-Besides providing several bind mounts for Docker socket, _procfs_ and journal directory, App tokens are required to ship data to the appropriate Monitoring Apps. Sematext Agent will gather data about running processes on the system, basic operating system metrics, machine/instance related information, and ship it to the Infra App token. It will also detect active containers and start collecting different container metrics such as memory usage, network I/O statistics, disk throughput, etc.
+Besides providing several bind mounts for Docker socket, _procfs_ and journal directory, App tokens are required to ship data to the appropriate Monitoring Apps. Sematext Agent will gather data about running processes on the system, basic operating system metrics, machine/instance-related information, and ship it to the Infra App token. It will also detect active containers and start collecting different container metrics such as memory usage, network I/O statistics, disk throughput, etc.
+
+By default, the US region receiver endpoints are used to ship data to Sematext Cloud. You can override receiver addresses by either passing `SERVER_BASE_URL` for metrics receivers, `LOGS_RECEIVER_URL` and `EVENTS_RECEIVER_URL` for log and event receivers respectively, or specify an alternative (`EU`) region via the `REGION` environment variable.
+
+**Note that if any of the** `*_URL` **environment variables are set, region-specific receiver endpoints are ignored.**
+
+### Updating/Upgrading Sematext Agent on Docker
+
+To update or upgrade the Sematext Agent to the latest version, you can pull the latest image and recreate the container:
+
+```bash
+docker pull sematext/agent:latest
+docker rm -f st-agent
+docker run -d  --restart always --privileged -P --name st-agent \
+-v /:/hostfs:ro \
+-v /sys/:/hostfs/sys:ro \
+-v /var/run/:/var/run/ \
+-v /sys/kernel/debug:/sys/kernel/debug \
+-v /etc/passwd:/etc/passwd:ro \
+-v /etc/group:/etc/group:ro \
+-e INFRA_TOKEN=<YOUR_INFRA_APP_TOKEN_HERE> \
+-e REGION=<US or EU> \
+sematext/agent:latest
+```
 
-By default, the US region receiver endpoints are used to ship data to Sematext Cloud. You can override receiver addresses by either passing `SERVER_BASE_URL` for metrics receivers, `LOGS_RECEIVER_URL` and `EVENTS_RECEIVER_URL` for log and event receivers respectively or specify an alternative (`EU`) region via `REGION` environment variable.
+### Uninstalling Sematext Agent on Docker
 
-**Note that if any of the** `*_URL` **environment variables are set, region specific receiver endpoints are ignored.**
+To remove the Sematext Agent container from a Docker host, run the following command:
+
+```bash
+docker ps -a --format '{{.Names}}' | grep -E "sematext-agent|st-agent" | xargs -r docker rm -f
```

Comment:
Orphan Vector and AA containers issue.

Reply:
@boratanrikulu any suggestion for this?

Reply:
I don't have a simple solution for this right now

Reply:
I have a solution that I tested and it works, but I need exact name of the docker image for: STA, Vector and AA. Basically it goes like this:

`docker.exe ps -a -q --filter "ancestor=sematext/<STA_IMAGE_NAME>:latest" --filter "ancestor=sematext/<VECTOR_IMAGE_NAME>:latest" --filter "ancestor=sematext/<AA_IMAGE_NAME>:latest" | xargs -r docker stop | xargs -r docker rm`